pkgs/tools/misc/lerpn/default.nix

```diff
@@ -0,0 +1,23 @@
+{ python3, lib, fetchFromGitea }:
```

Comment:
One item per line is preferred.

Reply:
Reasoning? nixpkgs-fmt is happy.

Reply:
Future diffs will be better-looking.
Adding a new input will add a new green line, removing an input will generate a new red line.
On the other hand, your method will add a new green and a new red line at each new modification.

Reply:
Ah this makes a lot of sense after seeing `nixpkgs-fmt` putting the commas at the *start* of the lines when I do this, too.

Thank you! It seems I'm a little rusty, but also [I've never had you before](https://github.com/NixOS/nixpkgs/pulls?q=reviewed-by%3AAndersonTorres+author%3Ackiee+), so hi! :P

(feel free to resolve this, leaving it open for visbility)